packages/InlineEditing/tests/InlineEditing.spec.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { render, screen, waitForElementToBeRemoved } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { ListBoxSingleStory } from "../stories/examples/ListBox";
+
+test("Inline editing / ListBox / open", async () => {
+  render(<ListBoxSingleStory />);
+
+  const triggerButton = screen.getByText(/oversight/i);
+  expect(triggerButton).toBeInTheDocument();
+
+  const regexOption1 = /Professional/i;
+  const regexOption2 = /Results Lite Contributor/i;
```

Comment:
You use these in all the tests - could move it up instead of redeclaring

Reply:
My opinion is that all tests should be atomic and should run independently, that's why I put all that is required for the test inside of it.